doc/source/user_guide/label_guide.rst

```diff
@@ -50,24 +54,28 @@ tell it to rename the variable name ``theta`` to ``$\theta$``, like so:
     @savefig label_guide_plot_posterior.png
     In [1]: az.plot_posterior(schools, var_names="theta", coords=coords, labeller=labeller, ref_val=5);
 
-You can see the labellers available in ArviZ at :ref:`their API reference page <labeller_api>`.
-Their names aim to be descriptive and they all have examples in their docstring.
-For further customization continue reading this guide.
+Additional Resources
+~~~~~~~~~~~~~~~~~~~~
+
+- For a list of labellers available in ArviZ, see the :ref:`the API reference page <labeller_api>`.
+- For common customization classes, see the ``arviz.labels`` module.
```

Comment:
The api reference has the docs for the `arviz.labels` module, the original last line tried to give a choice to readers. At this point they have seen how to use the `labeller` argument. Therefore, if they only want to do some more or less simple changes to the labels, they don't need to continue reading this guide, but they will probably need to read the api reference to find the class that suits their needs (which may be IdxLabeller instead of MapLabeller). Does this make sense? Maybe deleting this 2nd bullet point is enough already?

Also, how about using the `seealso` directive? https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-seealso

Reply:
Thank you for the hint with the `seealso` directive! This is what I was looking for.